public/js/app/stores/createApplicationStore.js

```diff
@@ -244,7 +244,7 @@ async function _filterOutUsedRounds(usedRoundsThisTerm, chosenSemester, koppsLas
   return (
     (thisTerm &&
       thisTerm.rounds &&
-      (await thisTerm.rounds.filter(r => !usedRoundsThisTerm.includes(r.ladokRoundId)).reverse())) ||
+      (await thisTerm.rounds.filter(r => !usedRoundsThisTerm.includes(r.applicationCodes[0])).reverse())) ||
```

Comment:
safe extracting of applicationCodes[0]

Reply:
Changed array to direct string. Now round.applicationCode will be used.